docs/DeveloperGuide.md

```diff
@@ -544,25 +544,25 @@ testers are expected to do more *exploratory* testing.
    2. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-### Adding a person
+### Adding an elderly
 
-1. Adding a person without a tag
+1. Adding an elderly without a tag
    1. Test case: `add i/S6516486H n/James Lim p/91234567 e/james.lim@hotmail.com a/432, Clementi East Ave 4, #13-42 c/7`<br>
-      Expected: A new person is added to the list. The details of the person are shown in the list.
+      Expected: A new elderly is added to the list. The details of the elderly are shown in the list.
 
-2. Adding a person with a tag
+2. Adding an elderly with a tag
    1. Test case: `add i/S1486256J n/Alice Tan p/98765432 e/alice.tan@gmail.com a/123, Jurong West Ave 6, #08-111 t/wheelchairBound c/7`<br>
-      Expected: A new person is added to the list. The details of the person are shown in the list and the "wheelchairBound" tag is shown.
+      Expected: A new elderly is added to the list. The details of the elderly are shown in the list and the "wheelchairBound" tag is shown.
 
-3. Adding a person with a duplicate NRIC
+3. Adding an elderly with a duplicate NRIC
    1. Test case: `add i/S6516486H n/Bernard Lim p/98375489 e/b.lim@me.com a/80, Lorong 4 Toa Payoh, #12-34 c/7`<br>
-      Expected: No new person is added. Error message is shown in the status bar.
+      Expected: No new elderly is added. Error message is shown in the status bar.
 
 ### Deleting an elderly
 
-1. Deleting an elderly while all people are listed in the `personList`.
+1. Deleting an elderly while all elderlies are listed in the `personList`.
```

Comment:
not sure if it should be all elderly or all elderlies

Reply:
Haha I wasn't sure also. I'll put it as elderly since I think it sounds more correct